src/gui/torrentcontentmodel.h

```diff
@@ -54,19 +56,22 @@ class TorrentContentModel: public QAbstractItemModel
     void updateFilesAvailability(const QVector<qreal> &fa);
     QVector<int> getFilePriorities() const;
     bool allFiltered() const;
-    virtual int columnCount(const QModelIndex &parent = QModelIndex()) const;
-    virtual bool setData(const QModelIndex& index, const QVariant& value, int role = Qt::EditRole);
+    int columnCount(const QModelIndex &parent = QModelIndex()) const override;
+    bool setData(const QModelIndex& index, const QVariant& value, int role = Qt::EditRole) override;
     TorrentContentModelItem::ItemType itemType(const QModelIndex& index) const;
     int getFileIndex(const QModelIndex& index);
-    virtual QVariant data(const QModelIndex& index, int role = Qt::DisplayRole) const;
-    virtual Qt::ItemFlags flags(const QModelIndex& index) const;
-    virtual QVariant headerData(int section, Qt::Orientation orientation, int role) const;
-    virtual QModelIndex index(int row, int column, const QModelIndex& parent = QModelIndex()) const;
-    virtual QModelIndex parent(const QModelIndex& index) const;
-    virtual int rowCount(const QModelIndex& parent = QModelIndex()) const;
+    QVariant data(const QModelIndex& index, int role = Qt::DisplayRole) const override;
+    Qt::ItemFlags flags(const QModelIndex& index) const override;
+    QVariant headerData(int section, Qt::Orientation orientation, int role) const override;
+    QModelIndex index(int row, int column, const QModelIndex& parent = QModelIndex()) const override;
+    QModelIndex parent(const QModelIndex& index) const override;
+    int rowCount(const QModelIndex& parent = QModelIndex()) const override;
     void clear();
     void setupModelData(const BitTorrent::TorrentInfo &info);
 
+    static bool useSystemFileIcons();
+    static void useSystemFileIcons(bool use);
```

Comment:
>Fine, I understand your point. But that is grammatically incorrect. I don't know how to keep it correct and satisfy that rule simultaneously.

Agree. But we write not English but C++ :) There is some difference. We can't have a proper English phrases everywhere. It's just Qt naming convention which we use too: setter name is `set<property_name>`, getter name is `<property_name>` (or `is<property_name>`/`has<property_name>` in case of bool type).
>enum class FileIcons

I think it's too much. Besides, where did the third value?